README.md

```diff
@@ -24,7 +24,7 @@ npm i @cloudflare/wrangler -g
 ```
 
 #### Specify binary site URL
-In case you need to store/mirror binaries on premise you will need to specify where wrangler should search for them by providing any of the following:
+In case you need to store/mirror binaries on premises you will need to specify where wrangler should search for them by providing any of the following:
```

Comment:
It's on premises. The other changes I made in the PR were just because I was already making a PR to change my personal pet peeve. https://brians.wsu.edu/2016/05/30/premise-premises/

Reply:
nice :)